src/GraphQL.Tests/Execution/VariablesTests.cs

```diff
@@ -109,7 +109,7 @@ public void executes_with_complex_input()
               fieldWithObjectInput(input: {a: ""foo"", b: [""bar""], c: ""baz""})
             }
             ";
-            var expected = "{ \"fieldWithObjectInput\": \"{\\\"a\\\":\\\"foo\\\",\\\"b\\\":[\\\"bar\\\"],\\\"c\\\":\\\"baz\\\"}\" }";
+            var expected = "{ \"fieldWithObjectInput\": \"{\\\"a\\\":\\\"foo\\\",\\\"b\\\":[\\\"bar\\\"],\\\"c\\\":\\\"baz\\\",\\\"d\\\":null}\" }";
```

Comment:
[Here is the matching tests in the JS project.](https://github.com/graphql/graphql-js/blob/master/src/execution/__tests__/variables.js#L227)  Shows only outputting the input values that are given.  So this is unexpected behavior.